examples/contracts-and-usage-policy/templates/UsageLoggingTemplates/USAGE_LOGGING_AGREEMENT_TEMPLATE.jsonld

```diff
@@ -17,7 +17,14 @@
             "@type": "ids:Duty",
             "ids:assigner": (idsc:PARTICIPANT_TEMPLATE),          // same value as ids:provider
             "ids:assignee": (idsc:PARTICIPANT_TEMPLATE),          // same value as ids:consumer
-            "ids:action": { "@id": "idsc:LOG" }
+            "ids:action": { 
+				"@id": "idsc:LOG",,
+				"refinement": [{    
+					"leftOperand": "idsc:LoggingEndpoint",  
+					"operator": "eq",  
```

Comment:
That's more a assignment, so idsc:DEFINES_AS?

Reply:
Here, since it is a URI too, maybe we could use the idsc:SAME_AS again.